polls/templates/polls/temp_login.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
+
+{% block body-block %}
+
+<div class="mt-5 text-center">
+  {{message}}
+</div>
```

Comment:
@najamF18 I don't think all users that would be coming for v1 testing would be so literate about inputting a mail in the URL, plus some of them would be on mobile which is second to impossible to edit URL.
How about we have a template view that has a single input and a button where they can input their mail, then from on click of the button we send get request to this temp login view and they get signed in?
I'm just thinking of simplicity and our expected user type.

Reply:
Sure. I was also thinking about that but I saw the conversation and did it this way. Anyways will resolve it